crates/libcontainer/src/namespaces.rs

```diff
@@ -39,6 +39,8 @@ static ORDERED_NAMESPACES: &[CloneFlags] = &[
     CloneFlags::CLONE_NEWNET,
     CloneFlags::CLONE_NEWCGROUP,
     CloneFlags::CLONE_NEWNS,
+    // CLONE_NEWTIME
+    CloneFlags::from_bits_retain(LinuxNamespaceType::Time as i32),
```

Comment:
Please add a comment stating that `CLONE_NEWTIME` does not exist in the `CloneFlags` type of the nix library.

```
// NOTE: CloneFlags in nix does not currently define CLONE_NEWTIME,
```

Also, it might be better to define a constant like the following somewhere:

```
const CLONE_NEWTIME_FLAG: CloneFlags =
    CloneFlags::from_bits_retain(LinuxNamespaceType::Time as i32);
```